test/ng/sceSpecs.js

```diff
@@ -464,6 +464,38 @@ describe('SCE', function() {
             '$sce', 'insecurl', 'Blocked loading resource from url not allowed by $sceDelegate policy.  URL: foo');
         }
       ));
+
+      describe('when the document base URL has changed', function() {
+        var baseElem = window.document.createElement('BASE');
+        var cfg = {whitelist: ['self'], blacklist: []};
+        baseElem.setAttribute('href', window.location.protocol + '//foo.example.com/path/');
+        beforeAll(function() {
+          window.document.head.appendChild(baseElem);
+        });
+        afterAll(function() {
+          window.document.head.removeChild(baseElem);
+        });
+        function expectAllowed($sce, url) {
```

Comment:
This helper (and `expectBlocked` below) seem redundant in this case. Let's remove them and inline the code.